.github/workflows/manual-publish.yml

```diff
@@ -35,10 +35,11 @@ jobs:
       - uses: ./.github/actions/build
         id: build
 
-      - uses: ./.github/actions/publish
+      - name: Publish package distributions to PyPI
+        if: ${{ inputs.dry_run == false }}
+        uses: pypa/gh-action-pypi-publish@release/v1
```

Comment:
FYI, the security recommendation is to have this in a dedicated job and avoid giving elevated OIDC privileges to transitive build dependencies.